### DISCUSSION TITLE
UX in mobile mode

### DISCUSSION THREAD

**Discussion by adyanth** [jesec/flood#345] [Ideas]
I would love to see the collapsible sidebar collapse when I click outside the sidebar rather than go hunting for the toggle button.

**Comment by jamauai** [comment]
Where's the toggle button?

**Reply by adyanth** [reply]
The 3 lines on top left?

**Reply by jamauai** [reply]
Sorry, looks like I had an older version installed.

**Comment by jesec** [comment]
That would add some potentially unnecessary complexities and may require logics that can't be handled by pure CSS, as you don't want that sort of behavior on desktop. 

I can take a look.

**Reply by adyanth** [reply]
Yes, that should only affect mobile view, maybe based on the viewport width combined with @media in CSS?

**Reply by jesec** [reply]
If I understand it correctly, you want "touch inside torrent list to dismiss sidebar". I don't think it is possible to implement that in CSS.

I think the straightforward solution would be a transparent overlay in the boundary of torrent list with `onClick` listener.

**Reply by adyanth** [reply]
In which case, as you noted, that would not be preferable on desktops where the real estate is not an issue.
